src/olympia/devhub/utils.py

```diff
@@ -314,8 +314,18 @@ def add_dynamic_theme_tag(version):
         Tag(tag_text='dynamic theme').save_tag(version.addon)
 
 
+def collect_existing_translations_from_addon(addon, properties):
```

Comment:
Could be me but `collect` doesn't tell me that this is doing a database query, `fetch` sounds a bit more "this does query some data, dude, don't overuse it" :man_shrugging: